<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color : #ffffff;
}</string>
   </property>
   <widget class="QPushButton" name="LeftParenthese">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color :#fff5cc;
	color : #6a777c;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
	transition : 0.4s;
}
QPushButton:pressed {
	background-color :#ffd11a;
	color : #ffffff;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button0">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>430</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Spartan</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color : #ffffff;
	color : #2e3436;
    border-radius : 20px; 
	box-shadow : 1px 1px 7px #2e3436;
	font-size : 14px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Del">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color :#ffe6ff;
	color : #6a777c;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
	transition : 0.4s;
}
QPushButton:pressed{
	background-color :#ff4dff;
	color : #ffffff;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>◄</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Memory3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Outfit</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border : none;
	padding-right :  15px ; 
	color : #6a777c;
	font-size : 11px
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Mod">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color :#fff5cc;
	color : #6a777c;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
	transition : 0.4s;
}
QPushButton:pressed {
	background-color :#ffd11a;
	color : #ffffff;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mult">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color :#ffe6ff;
	color : #6a777c;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
	transition : 0.4s;
}
QPushButton:pressed{
	background-color :#ff4dff;
	color : #ffffff;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Divide">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color :#ffe6ff;
	color : #6a777c;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
	transition : 0.4s;
}
QPushButton:pressed{
	background-color :#ff4dff;
	color : #ffffff;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Spartan</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color : #ffffff;
	color : #2e3436;
    border-radius : 20px; 
	box-shadow : 1px 1px 7px #2e3436;
	font-size : 14px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Display">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Outfit</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border : none;
	padding:15px;
	color :  #232829;
	font-size : 45px;
}</string>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Equal">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>380</y>
      <width>51</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color :#cc00cc;
	color : #fff;
	border-radius: 20px;
	font-size : 19px;
	box-shadow : 2px 2px 4px  #6a777c;
	transition : 0.5s;

}
QPushButton:pressed{
	background-color :#e600e6;
	color : #ffffff;
	border-radius: 20px;
	font-size : 19px;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Spartan</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color : #ffffff;
	color : #2e3436;
    border-radius : 20px; 
	box-shadow : 1px 1px 7px #2e3436;
	font-size : 14px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Memory2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Outfit</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border : none;
	padding-right :  15px ; 
	color :  #6a777c;
	font-size : 13px
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Button9">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Spartan</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color : #ffffff;
	color : #2e3436;
    border-radius : 20px; 
	box-shadow : 1px 1px 7px #2e3436;
	font-size : 14px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Subs">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color :#ffe6ff;
	color : #6a777c;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
	transition : 0.4s;
}
QPushButton:pressed{
	background-color :#ff4dff;
	color : #ffffff;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Spartan</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color : #ffffff;
	color : #2e3436;
    border-radius : 20px; 
	box-shadow : 1px 1px 7px #2e3436;
	font-size : 14px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Spartan</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color : #ffffff;
	color : #2e3436;
    border-radius : 20px; 
	box-shadow : 1px 1px 7px #2e3436;
	font-size : 14px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Square">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color :#fff5cc;
	color : #6a777c;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
	transition : 0.4s;
}
QPushButton:pressed {
	background-color :#ffd11a;
	color : #ffffff;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Spartan</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color : #ffffff;
	color : #2e3436;
    border-radius : 20px; 
	box-shadow : 1px 1px 7px #2e3436;
	font-size : 14px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color :#ffe6ff;
	color : #6a777c;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
	transition : 0.4s;
}
QPushButton:pressed{
	background-color :#ff4dff;
	color : #ffffff;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Dot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Spartan</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color : #ffffff;
	color : #2e3436;
    border-radius : 20px; 
	box-shadow : 1px 1px 7px #2e3436;
	font-size : 14px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Memory1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>53</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Outfit</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border : none;
	padding-right :  15px ; 
	color : #465053;
	font-size : 16px
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Button3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Spartan</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color : #ffffff;
	color : #2e3436;
    border-radius : 20px; 
	box-shadow : 1px 1px 7px #2e3436;
	font-size : 14px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Spartan</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color : #ffffff;
	color : #2e3436;
    border-radius : 20px; 
	box-shadow : 1px 1px 7px #2e3436;
	font-size : 14px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RightParenthese">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color :#fff5cc;
	color : #6a777c;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
	transition : 0.4s;
}
QPushButton:pressed {
	background-color :#ffd11a;
	color : #ffffff;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Flush">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color :#ffcc00;
	color : #ffffff;
    border-radius : 20px; 
	font-size : 14px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sign">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color :#fff5cc;
	color : #6a777c;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
	transition : 0.4s;
}
QPushButton:pressed {
	background-color :#ffd11a;
	color : #ffffff;
    border-radius : 20px; 
	font-size : 16px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>±</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Spartan</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color : #ffffff;
	color : #2e3436;
    border-radius : 20px; 
	box-shadow : 1px 1px 7px #2e3436;
	font-size : 14px;
	border : none;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
